content/learn/02.microcontrollers/03.analog-output/analog-output.md

```diff
@@ -13,6 +13,8 @@ In the graphic below, the green lines represent a regular time period. This dura
 
 ![](./pwm.gif)
 
+PWM is only available on digital output pins 3, 5, 6, 9, 10 and 11. The other digital pins cannot be used for PWM.
```

Comment:
```suggestion
On some microcontrollers PWM is only available on selected pins. Please consider the pinout diagram of your board to find out which ones you can use for PWM. They are denoted with a tilde sign (~).
```

Reply:
@sebromero yes this works great, thanks. Let's see if in a later stage, we can add visuals to explain this better.